src/index.css

```diff
@@ -24,3 +24,19 @@ body {
 .search-row input {
   margin-right: 10px;
 }
+
+.footer li {
+  list-style: none;
+}
+
+.card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
```

Comment:
The style of the cards looks really nice - like it.  Its very satisfying to create a very pleasing look.